front/src/components/boxs/todolist/TodolistBox.jsx

```diff
@@ -0,0 +1,158 @@
+import get from 'get-value';
+import { Component } from 'preact';
+import { Text } from 'preact-i18n';
+import { connect } from 'unistore/preact';
+import actions from '../../../actions/dashboard/boxes/todolist';
+import {
+  DASHBOARD_BOX_DATA_KEY,
+  DASHBOARD_BOX_STATUS_KEY,
+  GetTodolistStatus,
+  RequestStatus
+} from '../../../utils/consts';
+import style from './style.css';
+import dayjs from 'dayjs';
+
+const padding = {
+  padding: '1rem'
+};
+
+const BOX_REFRESH_INTERVAL_MS = 1 * 60 * 1000;
+
+const Task = ({ task, onClick }) => {
+  const icon = task.pending ? 'check-circle' : 'circle';
+  const textStyle = {
+    textDecoration: task.pending ? 'line-through' : undefined
+  };
+
+  return (
+    <div class={style.todolistTask}>
+      <div class="container">
+        <div class="row">
+          <span>
+            <i className={`fe fe-${icon}`} style={{ marginRight: '10px', cursor: 'pointer' }} onClick={onClick} />
```

Comment:
We usually don't use `className` attribute.
And I'm not fan of explicit styles in HTML element, please prefer class usage (`mr-2 cursor-pointer`).